### PR TITLE
Update Carnegie-Institution-for-Science.xml

### PR DESCRIPTION
**Pull request by tamoor-cliqz**

### CODE REVIEW

**Comment by mananaBanana** [comment]
mail.ciw.edu is unreachable. Need to comment out the rule for the same. Should I take this up?

**Comment by jeremyn** [comment]
@tamoor-cliqz Are you still interested in working on this?

**Comment by J0WI** [comment]
cc @luciancor

**Comment by luciancor** [comment]
@J0WI fixed!

File: src/chrome/content/rules/Carnegie-Institution-for-Science.xml [hunk 1]
```diff
@@ -33,12 +34,11 @@
 	<target host="vpn.carnegiescience.edu" />
 	<target host="www.carnegiescience.edu" />
 
-	<target host="hazen.gl.ciw.edu" />
+	<!-- <target host="hazen.gl.ciw.edu" /> Expired Certificate
```
Comment (J0WI) [inline src/chrome/content/rules/Carnegie-Institution-for-Science.xml]: I get a mismatch. Please also move this information on the top comment block.

**Comment by luciancor** [comment]
Updated the comment

**Pull request merged**